### ISSUE TITLE
where final output has no softmax

### ISSUE THREAD

**Issue by shisi2015** [Kyubyong/transformer#58]
self.logits = tf.layers.dense(self.dec, len(en2idx))

**Comment by DaoD** [comment]
For training, the softmax_cross_entropy_with_logits will compute softmax.
For predicting, the argmax result is the same before or after using softmax.

**Comment by policeme** [comment]
@DaoD can you describe this more detail? thx

**Comment by policeme** [comment]
thx, I understand it.
